pages/common/git-daemon.md

```diff
@@ -0,0 +1,16 @@
+# git daemon
+
+> A really simple server for Git repositories.
+> More information: <https://git-scm.com/docs/git-daemon>.
+
+- Launch a Git daemon with a whitelisted set of directories:
+
+`git daemon --export-all {{path/to/directory1}} {{path/to/directory2}}`
+
+- Launch a Git daemon with a base directory and allow pulling from all directories that look like Git repos:
+
+`git daemon --base-path={{path/to/directory}} --export-all --reuseaddr`
+
+- Launch a Git daemon with a base directory with allowed push access and verbosity:
+
+`git daemon --base-path={{path/to/directory}} --export-all --enable=receive-pack --reuseaddr --informative-errors --verbose`
```

Comment:
wdyt @marchersimon?
```suggestion
- Launch a Git daemon, verbosely printing log messages and allowing Git clients to push:

`git daemon --enable=receive-pack --informative-errors --verbose`
```

Reply:
Sounds good, however `push` alone sounds a bit weird. How about ` allowing Git clients to push commits`? (On the other side you can push more than just commits, right?)

Reply:
Yes, you can push more than commits — you can also push tags and branches.